libtsuba/include/tsuba/RDGSlice.h

```diff
@@ -20,6 +20,8 @@ namespace tsuba {
 class RDGManifest;
 class RDGCore;
 
+enum class NodeEdge { kNode = 10, kEdge, kNeitherNodeNorEdge };
```

Comment:
Is this still needed? I saw that you saved it from the PropertyCache header but I'm not sure why.

Reply:
There is code in RDGSlice.cpp that use it.  The code looks useful.

Reply:
But I moved the definition from the .h file into the .cpp file.  That seems worthwhile.